dockers/docker-orchagent/base_image_files/swssloglevel

```diff
@@ -7,4 +7,37 @@ if [ -t 1 ] ; then
     DOCKER_EXEC_FLAGS+="t"
 fi
 
-docker exec -$DOCKER_EXEC_FLAGS swss swssloglevel "$@"
+function help()
+{
+    echo -e "Usage: $0 -n [0 to $(($NUM_ASIC-1))] [OPTION]... " 1>&2; exit 1;
+}
+
+DEV=""
+PLATFORM=`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`
```

Comment:
i think we can use the env file to reduce the overhead of this call.

Reply:
thanks ! updated to use the env variables.

Reply:
@lguohan, Could you confirm if the changes are ok.